cmd/manager/fips.go

```diff
@@ -1,15 +0,0 @@
-// +build fips_enabled
```

Comment:
This has been deleted entirely and I can't see it has been moved anywhere, what happens to fips enablement as a result?

Reply:
Thanks. Run `make`  to generate this file and it's added now.